docs/quickstart.md

```diff
@@ -95,15 +95,54 @@ root = "/var/lib/firecracker-containerd/containerd"
 state = "/run/firecracker-containerd"
 [grpc]
   address = "/run/firecracker-containerd/containerd.sock"
-[proxy_plugins]
-  [proxy_plugins.firecracker-naive]
-    type = "snapshot"
-    address = "/var/lib/firecracker-containerd/naive-snapshotter.sock"
+[plugins]
+  [plugins.devmapper]
+    pool_name = "fc-dev-thinpool"
+    base_image_size = "10GB"
+    root_path = "/var/lib/firecracker-containerd/snapshotter/devmapper"
 
 [debug]
   level = "debug"
 EOF
 
+# Setup device mapper thin pool
+sudo mkdir -p /var/lib/firecracker-containerd/snapshotter/devmapper
+cd /var/lib/firecracker-containerd/snapshotter/devmapper
+DIR=/var/lib/firecracker-containerd/snapshotter/devmapper
+POOL=fc-dev-thinpool
+
+if [[ ! -f "${DIR}/data" ]]; then
+sudo touch "${DIR}/data"
```

Comment:
(multiple occurrences) Can you indent inside the `if` blocks?

Reply:
Done